src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
```

Comment:
Ensure that the 'table' element exists in the DOM before querying its children. If the table is not present, this will cause a runtime error.